lib/bemhtml/runtime/match.js

```diff
@@ -216,7 +227,15 @@ Match.prototype.exec = function exec(context) {
   this.bemhtml.match = oldMatch;
   this.restoreDepth(save);
 
+  var e;
+  if (this.thrownError !== null) {
+    e = this.thrownError;
```

Comment:
NonBlocker: btw, you can reduce 1 line of code with moving this to `var`. And also you can replace `this.thrown` in `if` with just `e`.